docs/config.md

```diff
@@ -0,0 +1,156 @@
+# Talos CCM Configuration
+
+## Overview
+
+The Talos CCM is a Kubernetes controller that manages the lifecycle of Talos nodes in a Kubernetes cluster.
+
+In scope of the Talos CCM, the following features are implemented:
+* Initialize nodes in a Kubernetes cluster, set [well-known labels and annotations](https://kubernetes.io/docs/reference/labels-annotations-taints/).
+* Label and annotate nodes based on the transformation rules.
+* Approve and sign the [kubelet certificate signing request](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#kubernetes-signers) for Talos nodes.
+
+Result of kubernetes node object after Talos CCM initialization:
+
+```yaml
+apiVersion: v1
+kind: Node
+metadata:
+  annotations:
+    ...
+    # Set annotations based on transformation rules
+    talos.dev/instance-id: "id-e8e8c388-5812-4db0-87e2-ad1fee51a1c1"
```

Comment:
Should there also be an omni instance-id or will they be the same?

Reply:
this is just a key/value string, You can set everything that you want. In this example - `talos.dev/instance-id: "id-{{ .InstanceID }}"` there InstanceID - is cloud instance ID

Reply:
I think my main question is if this should be `talos.dev/` or an omni identifier. Will the ID be the same in this label as it shows up in Omni?

Reply:
Oh i see, i renamed it, to not confuse people ;) `custom-annotation/instance-id`